pvactools/lib/top_score_filter.py

```diff
@@ -137,6 +137,19 @@ def __init__(
             anchor_probabilities[length] = probs
         self.anchor_probabilities = anchor_probabilities
 
+        mouse_anchor_positions = {}
+        for length in [8, 9, 10, 11]:
+            base_dir = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..'))
+            file_name = os.path.join(base_dir, 'tools', 'pvacview', 'data', "mouse_anchor_predictions_{}_mer.tsv".format(length))
+            values = {}
+            with open(file_name, 'r') as fh:
+                reader = csv.DictReader(fh, delimiter="\t")
+                for line in reader:
+                    allele = line.pop('Allele')
+                    values[allele] = line
```

Comment:
Another option here is to convert the strings into boolean values. You can do this via the `ast` library.
```
>>> import ast
>>> ast.literal_eval("False")
False
>>> type(ast.literal_eval("False"))
<class 'bool'>
```
```suggestion
                    values[allele] = [ast.literal_eval(v) for v in line]
```
However, this only works with "True"/"False", not "TRUE"/"FALSE" so you'd need to change the values in your TSV.

The advantage is that you can then change line 164 in `pvactools/lib/run_utils.py` to be `if val` instead of having to do a string comparison.

Nothing wrong with your current implementation; just wanted to mention an alternate approach.